atlasdb-cassandra-tests/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraTestSuite.java

```diff
@@ -33,23 +32,20 @@
 
 @RunWith(Suite.class)
 @SuiteClasses({
```

Comment:
you probably did this just for testing your changes quickly (I do this too) and forgot to revert this section, I will add the other tests back in to the suite